package.json

```diff
@@ -138,7 +138,6 @@
     "@skyux/layout": "7.0.0",
     "@skyux/lookup": "7.0.0",
     "@skyux/theme": "7.0.0",
-    "clipboard-polyfill": "2.8.6",
```

Comment:
We can safely remove `clipboard-polyfill` since the native `navigator.clipboard.copyText` is supported in all modern browsers.